### PR TITLE
like page and functionalities

### PR DESCRIPTION
**Pull request by jeetsdev**
## Added like page and functionalities

- Users can like a particular video 
- User can see all the liked videos 
- User can remove the particular video from liked videos.

[Preview]()

### CODE REVIEW

File: src/components/Cards/LikesCard/LikesCard.jsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+export const LikesCard = ({ video }) => {
+    const { thumbnail, title, description, creatorAvatar, creator
+        , views } = video;
+
+    const { removeFromLike } = useLike();
+
+    // Shorting title and description here
+    const shortDescr = description.split(" ").slice(0, 20).join(" ");
+    const shortTitle = title.split(" ").slice(0, 8).join(" ");
```
Comment (Pratik1005) [inline src/components/Cards/LikesCard/LikesCard.jsx]: you can create a function to short strings

**Review by Pratik1005 — APPROVED** [review]
Rest looks good

**Review by rahulmistry751 — APPROVED** [review]
Good UI and folder structure implemented

File: src/components/Cards/LikesCard/LikesCard.jsx (hunk 1 above)
Comment (jeetsdev) [reply src/components/Cards/LikesCard/LikesCard.jsx]: Yeah sure, Thanks for the suggestion.

**Pull request merged**